Example/Example/ExampleViewController.swift

```diff
@@ -7,11 +7,45 @@
 //
 
 import UIKit
+import Ably
 
 class ExampleViewController: UIViewController {
 
+    private var client: ARTRealtime!
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        connectToAbly()
+    }
+
+    private func connectToAbly() {
+        /* Set up a Realtime client that authenticates with the local web server auth endpoint */
+        let options = ARTClientOptions()
+        options.authUrl = NSURL(string: "http://localhost:4567/auth")
```

Comment:
Apologies, I realise the other browser demos use `authUrl`, but in almost all cases we'd never get a token like this from a mobile app.  The other auth demos use `authUrl` simply because the "app" is a browser app.  So in most cases users would build a login system with their website, and as such, the `authUrl` would send the cookie in the request and the server as a result could determine the identity of that user.  In the case of a mobile app, this is never the case and a cookie is almost never sent.  So instead, in mobile apps / desktop apps, we expect `authCallback` to be used and the logic to obtain a token from the dev's servers would be done by the app as the app would know how to authenticate / confirm it's logged in etc.  We need to change this please so that the app uses `authCallback` which calls a method such as `obtainTokenRequest` which in turns makes an HTTP request to the server.  The comment then with this `obtainTokenRequest` method could be something like `/* For simplicity, in this example we send a HTTP request to the server to obtain a token request. Typically an app will have authenticated a user and will thus be responsible for in turn authenticating with the server to obtain a token request for the currently logged in user */`